tsconfig.json

```diff
@@ -12,8 +12,9 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "jsx": "preserve"
+    "jsx": "preserve",
+    "baseUrl": "."
```

Comment:
I tried to set this to `"baseUrl": ".."` in `worker/tsconfig.json` but it did not work.

Perhaps you can't have a baseUrl in a parent directory?